source/administration/config-settings.rst

```diff
@@ -290,34 +290,89 @@ Set policy on who can invite others to a team using "Invite New Member" to invit
 | This feature's ``config.json`` setting is ``"RestrictTeamInvite": "all"`` with options ``all``, ``team_admin`` and ``system_admin`` for above settings respectively. |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-Enable public channel management permissions for
+Enable public channel creation for
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Restrict the permission levels required to create, delete, rename, and set the header or purpose for public channels. The last member of a public channel has the ability to delete the channel regardless of their permission level.
+Restrict the permission levels required to create public channels.
 
-**All team members**: Channel management permissions for public channels are enabled for all users.
+**All team members**: Channel creation permissions for public channels are enabled for all users.
 
-**Team and System Admins**: Channel management permissions for public channels are restricted to Team and System Admins.
+**Team and System Admins**: Channel creation permissions for public channels are restricted to Team and System Admins.
```

Comment:
similarly here to `Public channel creation restricted to Team and System Admins` --> or `Restrict public channel deletion to ...`.